autoemulate/experimental/emulators/base.py

```diff
@@ -51,16 +48,16 @@ def model_name(cls) -> str:
         return cls.__name__
 
     @abstractmethod
-    def _predict(self, x: TensorLike) -> OutputLike:
+    def _predict(self, x: TensorLike, with_grad: bool = False) -> OutputLike:
```

Comment:
Given there is the option to set_grad_enabled, perhaps we do not need to add the the `_predict()` API? This might be helpful so that subclassing would not need to consider this by default.